zerver/lib/events.py

```diff
@@ -313,6 +313,9 @@ def fetch_initial_state_data(user_profile: UserProfile,
     if want('zulip_version'):
         state['zulip_version'] = ZULIP_VERSION
 
+    if want('zulip_feature_level'):
+        state['zulip_feature_level'] = FEATURE_LEVEL
```

Comment:
I think this probably belongs in the `zulip_version` want block -- we don't need a separate category for this.